packages/api/src/controllers/stream.ts

```diff
@@ -743,7 +752,8 @@ app.get("/playback/:playbackId", authorizer({}), async (req, res) => {
   });
   if (
     !stream ||
-    ((stream.userId !== req.user.id || stream.deleted) && !req.user.admin)
+    ((stream.userId !== req.user.id || stream.deleted) && !req.user.admin) ||
+    stream.projectId !== req.project?.id
```

Comment:
I would looooove to have that helper that checks all this on a resource and we don't have to keep repeating it 😭

Reply:
Yeah i was trying to do it here but was too bogged down with addressing each usage and making sure it worked. I'll file a ticket as a follow up to try and make this a generic helper.